test/transform-spec.ts

```diff
@@ -45,18 +45,28 @@ describe("transform", () => {
 
     const ret = transformer.transform(data);
 
-    expect(ret.isRight).to.be.true;
-    const out = ret.get();
+    expect(ret.isRight()).to.be.true;
+    const out = ret.getRight();
     expect(out instanceof C1).to.be.true;
     expect(out.name).to.equal('asd');
     expect(out.count).to.equal(2);
     expect(out.$orig).to.be.undefined;
     expect(out.id_).to.be.undefined;
   });
 
+  it("should transform undefined", () => {
```

Comment:
Yeah -- I'm surprised the linter didn't catch that here, since it caught it elsewhere.. maybe spec files are being filtered out or something..